.github/workflows/benchmark.yml

```diff
@@ -22,11 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          default: true
+      - uses: dtolnay/rust-toolchain@stable
```

Comment:
Do we prefer this action going further? Should we change the rest of the workflows? What are the advantages?

Reply:
Hehe I knew this question was going to come up. The last discussion we had about it was https://prisma-company.slack.com/archives/C4GCG53BP/p1669891725070999 — it seems like a better overall way to set up the toolchain, but it doesn't make a huge difference. I'd also prefer to standardize, so I'm open to making the switch everywhere or reverting the change in this PR right away, but I'm fine with doing it progressively too. What do you think?

Reply:
I'll make the change in a separate PR.

Reply:
WIP at https://github.com/prisma/prisma-engines/pull/3916